staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go

```diff
@@ -218,6 +218,10 @@ func (rl *respLogger) Log() {
 		}
 	}
 
+	if plName := request.GetPriorityLevelNameTruncated(rl.req.Context()); plName != "" {
+		keysAndValues = append(keysAndValues, "priorityLevel", plName)
```

Comment:
Multiple FlowSchemas can be associated with a single priorityLevel - so it may not be possible to get to the FlowSchema name from a given priority-level. 

One option is to add both `flowSchema` and `priorityLevel` into httplog, thoughts?

Reply:
Yes, the FlowSchema implies the priority level and not the other way around (for a given configuration), so if you want to log only one then it should be the FlowSchema.

Reply:
I was thinking on priorityLevel as this is actually what matters when we debug high latency calls.
Adding only flowSchema will make such debugging harder, but I'm happy to add flowSchema as well.

Reply:
> so if you want to log only one then it should be the FlowSchema.

I guess in most cases we would want to know the priorityLevel - if we add only `flowSchema` then the operator will need to run `kubectl` command and find the related `priorityLevel`, which may be cumbersome. I am +1 for adding both.

Reply:
FlowSchema matters, for at least two reasons:
1. Helping explain _why_ the particular priority level applied, and
2. the choice of flow distinguisher matters.

I am not enthused about making the httplog message longer.  Another consideration might be how much information can be gotten by an investigator who does not have access to the cluster config (either at all, or because it is no longer the current config).  Perhaps the logic that digests the config should be sure to write it at the same V level as the httplog message (which is 3, if I recall correctly)?

Reply:
I think this is worth making the message longer. I agree it's ideal to see something like  `APF(FS=<name> PL=<name> D=<value>)` to keep it compact. Log search is much easier if we include the PL.

Reply:
@lavalamp , what do you mean "D=<value>" to refer to?  As long as I am wondering, I suppose I wonder about the chosen width and extra latency.

Reply:
+1 for keeping this as part of httplog. It simply makes debugging easier.

httplog doesn't support anything more than keys and values, but as suggested by @MikeSpreitzer we can use shorter identifiers like 'pl' for priorityLevel and 'fs' for flowSchema.

Reply:
I added FlowDistinguisher and added APF_ prefix. PTAL